Makefile

```diff
@@ -191,36 +199,43 @@ info:
 	$(info pkgconfigdir = $(value pkgconfigdir))
 	$(info ------------------------------------)
 	@true
-.PHONY: info
+info-backends:
+	$(if $(OCCA_ON),\
+	  $(info libCEED: OCCA backend enabled [OCCA_DIR=$(OCCA_DIR)]))
+	$(if $(MAGMA_ON),\
+	  $(info libCEED: MAGMA backend enabled [MAGMA_DIR=$(MAGMA_DIR)]))
```

Comment:
How about a one-line `Building 'lib' with optional backends: OCCA MAGMA`? This would also help identify the target, as would normally be done automatically
```
make: Nothing to be done for 'lib'.
```
And since we expect more backends, the backend-specific part of a makefile should be contained. Like add to a list `BACKEND_PACKAGES += OCCA` instead of reproducing `OCCA_ON` and `MAGMA_ON` in multiple places in the makefile.

Reply:
I wanted to have the path displayed as well - I find that useful.

Feel free to make further improvements, in this PR or later.

Reply:
Not quite what I was going for but I'm okay with it.